bears/js/HappinessLintBear.py

```diff
@@ -19,6 +20,13 @@ class HappinessLintBear:
     ASCIINEMA_URL = 'https://asciinema.org/a/80714'
     CAN_DETECT = {'Syntax'}
 
-    @staticmethod
-    def create_arguments(filename, file, config_file):
+    @classmethod
+    def create_arguments(cls, filename, file, config_file,
+                         use_spaces: bool=False):
+        if use_spaces:
+            raise ValueError(
+                ('"use_spaces=True" is incompatible with {0}, '
+                 'set it to false'+re.escape('.')).format(cls.name)
```

Comment:
why do you do `re.escape`? :O This was intended for the tests below

Reply:
You said that the "." Can prove to misbehave somtimes so I......

Reply:
Yeah but only for the tests, as `self.assertRaisesRegex` is checking against a regex pattern, this is the wrong place

Reply:
okk got you.